marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/JobControllerTest.java

```diff
@@ -33,6 +34,7 @@
     JobController jobController;
 
     private final static String STEP_RESPONSES_PATH = "/api/jobs/stepResponses";
+    private final static String GET_MATCHING_VALUES_PATH = "/api/jobs/stepResponses/facet-values";
```

Comment:
Seeing "stepResponses" and "facet-values" begs the question - why is one camelCase and the other is worm-case? I'd think "facetValues" is better for consistency.